src/IssueViz.UnitTests/Editor/Common/CommonTaggerProviderTestsBase.cs

```diff
@@ -0,0 +1,95 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Tagging;
+using Moq;
+
+namespace SonarLint.VisualStudio.IssueVisualization.UnitTests.Editor.ErrorTagging
+{
+    /// <summary>
+    /// Common tests that apply to our TaggerProviders
+    /// </summary>
+    public abstract class CommonTaggerProviderTestsBase
```

Comment:
There will be enough variation between the tagger providers that it's not worth having a common base class for them. However, we do expect some common behaviours so we can have this set of common tests.